File: CarRental/ClientApp/app/viewOrder.ts
export class ViewOrder {
    constructor(
        public id?: number,
        public userId?: number,
        public userLastName?: number,
        public userFirstName?: string,
        public carId?: number,
        public carMake?: string,
        public carModel?: string,
        public carRegistrationNumber?: string,
        public startDate?: Date,
        public finalDate?: Date) { }
}

File: CarRental/Core/Models/ViewOrder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CarRental.Core.Models
{
    public class ViewOrder
    {
        public int Id { get; set; }
        public int UserId { get; set; }

        public string UserFirstName { get; set; }

        public int CarId { get; set; }
        public string CarMake { get; set; }

        public string CarRegistrationNumber { get; set; }

        public DateTime StartDate { get; set; }
        public DateTime FinalDate { get; set; }
    }
}
